docs/api-irma-server.md

```diff
@@ -182,30 +184,64 @@ Behaves exactly the same as the [delete endpoint for requestors](#delete-session
 
 ### `GET /irma/session/{clientToken}/status`
 Behaves exactly the same as the [status endpoint for requestors](#get-session-requestortoken-status), but uses the [client token
-from the `sessionPtr`](#post-session) instead of the requestor token.
+from the `sessionPtr`](#post-session) instead of the requestor token. For frontend libraries, this endpoint is deprecated.
+Please use the [frontend status endpoint](#get-irmasessionclienttokenfrontendstatus) instead.
 
 ---
 
 ### `GET /irma/session/{clientToken}/statusevents`
 Behaves exactly the same as the [statusevents endpoint for requestors](#get-session-requestortoken-statusevents), but uses the
-[client token from the `sessionPtr`](#post-session) instead of the requestor token.
+[client token from the `sessionPtr`](#post-session) instead of the requestor token. For frontend libraries this endpoint is deprecated.
+Please use the [frontend statusevents endpoint](#get-irmasessionclienttokenfrontendstatusevents) instead.
 
 ---
 
+## API reference `/irma/frontend/` endpoints
+The frontend endpoints are exclusively meant for [frontend libraries](irma-frontend.md) to communicate with the IRMA server.
+Frontends need the information from the `frontendRequest` in order to use these endpoints. The `frontendRequest` is received
+along with the `sessionPtr` from the [`POST /session`](#post-session) requestor endpoint.
+Just like the [other `/irma` endpoints](#api-reference-irma-endpoints), the `/irma/frontend` endpoints of your IRMA server
+have to be publicly reachable from the internet.
+
+To make sure these endpoints can only be accessed by frontends, requests should be done with an
+additional `Authorization` HTTP request header. The expected value for this request header is the `authorization`
+token in the `frontendRequest`.
+
+The frontend endpoints in this version of the IRMA server implement frontend protocol version 1.1.
+
+### `GET /irma/session/{clientToken}/frontend/status`
+Retrieve the current [session status](https://godoc.org/github.com/privacybydesign/irmago/server#Status), and additional information
+being relevant for that session status, as a JSON string.
```

Comment:
This endpoints returns not a string but an object.

Reply:
I now I understand what you meant with JSON string. I interpreted it as a string, containing JSON. That goes wrong at some other places too then, so I'll do a full ctrl-f on JSON string.